tests/dummy/app/index.html

```diff
@@ -16,6 +16,12 @@
   </head>
   <body>
     {{content-for "body"}}
+    <div id="qunit"></div>
+    <div id="qunit-fixture">
+      <div id="ember-testing-container">
+        <div id="ember-testing"></div>
+      </div>
+    </div>
```

Comment:
Should be removed, this should only be present in tests/index.html

Reply:
This was a mistake, and I already removed it.